docs/scripts/generate_plugin_docs.py

```
@@ -782,7 +816,9 @@ def _get_plugin_sort_key(self, plugin_tuple):
                 return (0, -repo_info.get("stars", 0))
         return (0, -repo_info.get("stars", 0) if repo_info else 0)
 
-    def generate_plugins_ecosystem_rst(self, all_plugins: List[Plugin]) -> str:
+    def generate_plugins_ecosystem_rst(
```

Ruff (0.15.11)

[warning] Too many branches (13 > 12)

(PLR0912)